keepassxc-browser/content/observer-helper.js

```diff
@@ -270,3 +282,17 @@ kpxcObserverHelper.ignoredNode = function(target) {
 
     return false;
 };
+
+// Traverses all children, including Shadow DOM elements
+const traverseChildren = function(target, children) {
+    for (const child of target.childNodes) {
+        if (child.nodeName === 'INPUT' && child.type !== 'hidden' && !child.disabled ) {
```

Comment:
If the child is hidden or disabled, we should stop traversing down (ie, return).

Reply:
Fixed. Also added one `continue` after adding inputs to the list.